roxie/ccd/ccdquery.cpp

```diff
@@ -494,6 +498,31 @@ void QueryOptions::setFromContext(const IPropertyTree *ctx)
     if (ctx)
     {
         // Note: priority cannot be set at context level
+        // b/c this is after activities have been created, but we could
+        // dynamically adj priority in the header activityId before sending
+        int tmpPriority;
+        updateFromContext(tmpPriority, ctx, "@priority", "_Priority");
+        if (tmpPriority > 1)
```

Comment:
Would be clearer to have some constants for MinQueryPriority and MaxQueryPriority and compare against those.  (Comparing > 1 would be clearer to compare with > 2, since that is leaking the next test into the bounds test).